README.md

```diff
@@ -178,7 +178,7 @@ version that is still in development, add
 For example:
 ```
 make build \
-  TAG=soroban-dev \
+  TAG=future \
```

Comment:
noticed we likely want to change the makefile target for `build-soroban-dev` to be consistent naming also, it still refs `soroban-dev`
https://github.com/stellar/quickstart/blob/leighmcculloch-patch-1/Makefile#L41

Reply:
oh nvm, you did this change in https://github.com/stellar/quickstart/pull/551/files